components/director/internal/domain/tenant/repository.go

```diff
@@ -87,13 +95,20 @@ func (r *pgRepository) ExistsByExternalTenant(ctx context.Context, externalTenan
 func (r *pgRepository) List(ctx context.Context) ([]*model.BusinessTenantMapping, error) {
 	var entityCollection EntityCollection
 
-	conditions := repo.Conditions{
-		repo.NewEqualCondition(statusColumn, Active),
+	persist, err := persistence.FromCtx(ctx)
+	if err != nil {
+		return nil, errors.Wrap(err, "while fetching persistence from context")
 	}
 
-	err := r.listerGlobal.ListGlobal(ctx, &entityCollection, conditions...)
+	prefixedFields := strings.Join(str.PrefixStrings(tableColumns, "t."), ", ")
+	query := fmt.Sprintf(`SELECT DISTINCT %s, ld.tenant_id IS NOT NULL AS %s
```

Comment:
It was agreed with our UI team to sort the tenants firstly by `inUse` field, and then by `name`.